docs/guides/sass.md

```diff
@@ -0,0 +1,126 @@
+- [Global flags](#global-flags)
+- [Feature flags](#feature-flags)
+- [Optimizing your Sass builds](#optimizing-your-sass-builds)
+- [FAQ](#faq)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- prettier-ignore-end -->
+
+## Overview
+
+The `carbon-components` package ships all of the styles for the Carbon Design
+System as Sass files in the `scss` folder. You can import this file directly
+through either of the following paths:
+
+```scss
+# Specifying node_modules directly
+@import 'node_modules/carbon-components/scss/<path-to-file>';
+
+# With webpack
+@import '~carbon-components/scss/<path-to-file>';
+
+# With sass config setup to include `node_modules` in paths
+@import 'carbon-components/scss/<path-to-file>';
+```
+
+There are two folders in this `scss` folder:
+
+- `components`: which contain component-specific styles and mixins
+- `globals`: which contain files that effect global settings like color, type,
+  grid, and more
+
+To quickly get started, you can import `styles.scss` which contains all of the
+styles for the Carbon Design System. You can import this using the following
+path:
+
+```scss
+@import 'carbon-components/scss/globals/scss/styles.scss';
+```
+
+_Note: the `styles.scss` will include all styles for the Carbon Design System,
+even if you are not using all of its components, to learn how to optimize this
+import check out [Optimizing your Sass builds](#optimizing-your-sass-builds)._
+
+If you would like to a see a full overview of the functionality we ship in Sass,
+in particular our public API, you should checkout our published
+[SassDoc](../../packages/components/docs/sass.md).
+
+## Global flags
+
+The Carbon Design System sass setup specifies a number of global flags that you
+can configure before importing Carbon's sass files to enable or disable
+different behaviors. To enable these flags, you will need to set them before
+importing any styles from Carbon. For example:
+
+```scss
+$css--reset: false;
+@import 'carbon-components/scss/globals/scss/styles.scss';
+```
+
+For a full reference of flags, see the table below.
+
+| Global flag       | Description                                                          | Default value |
+| ----------------- | -------------------------------------------------------------------- | ------------- |
+| `$css--font-face` | Includes the font-face mixins for the current font family (IBM Plex) | `true`        |
+| `$css--helpers`   | Includes classes and utilities that are commonly used by components  | `true`        |
+| `$css--body`      | Sets a top-level reset, type style, and color for the `<body>` tag   | `true`        |
+| `$css--use-layer` | Enables use of box-shadow in `layer()` helpers                       | `true`        |
+| `$css--reset`     | Includes a top-level CSS Reset                                       | `true`        |
+
+## Feature flags
+
+The Carbon Design System takes advantage of feature flags to conditionally
```

Comment:
Is there a list of what all of the available feature flags are at a given time?

Reply:
Good question! It should be showing up in our generated sassdoc but I'm not seeing it, let me bug @mattrosno to figure out why 😂

Reply:
On it!